docker-compose/docker-compose.yaml

```diff
@@ -491,6 +491,7 @@ services:
     image: 'index.docker.io/sourcegraph/cadvisor:insiders@sha256:3544dba5cd99c429ac62f4773454c8406abe126ba340f34e457b38d828c63395'
     cpus: 1
     mem_limit: '1g'
+    privileged: true
```

Comment:
I suppose some users would be worried with privileged (root) container, anything we can do to address future concern?

Reply:
Maybe add a comment saying it can be set to false, in case anyone has concerns? More likely they'd just disable cadvisor entirely but it can't hurt.